### PR TITLE
Update Sponsors.md

### PR DESCRIPTION
**Pull request by kingherc**

### CODE REVIEW

File: Sponsors.md
Comment (kingherc) [inline Sponsors.md:17]: I did not know if we want to remove these, so I left them, but removed specific amounts. I thought the amounts may be old and not up-to-date, so I just mention they should get into contact with us.

File: Sponsors.md
Comment (kingherc) [inline Sponsors.md:76]: I was not sure this section is very useful so I removed it.

File: Sponsors.md
Comment (kingherc) [inline Sponsors.md:87]: I made this as a sub-section of "Other ways to sponsor", as I am not sure we can vet OpenCollective sponsors (anyone can contribute there, right?).

File: Sponsors.md
Comment (mzampetakis) [inline Sponsors.md:31]: Maybe we could keep this more generic by using the amount of money spEnt/cost of the SWAG.
E.g. 100 € per meetup (leaflets, thank you message at meetup and social media).

File: Sponsors.md
Comment (mzampetakis) [inline Sponsors.md:43]: Isn't this a SWAG too? maybe we could combine with the previous section (SWAG). What do you think?

File: Sponsors.md
Comment (mzampetakis) [inline Sponsors.md:45]: I like this!

File: Sponsors.md
Comment (mzampetakis) [inline Sponsors.md:84]: Can't we replace `info @t devstaff d0t gr` with [info@devstaff.gr](mailto:info@devstaff.gr) ?

**Review by mzampetakis — CHANGES_REQUESTED** [review]
Great work @kingherc ! 👏 👏 👏
I 've added some comments. I would also love to see some feedback from other members as well!

File: Sponsors.md
Comment (kingherc) [reply Sponsors.md]: Done!

File: Sponsors.md
Comment (kingherc) [reply Sponsors.md:18]: Done

File: Sponsors.md
Comment (kingherc) [reply Sponsors.md]: Attempted something

File: Sponsors.md
Comment (kingherc) [reply Sponsors.md]: Tried EUR amounts instead. Please review

File: Sponsors.md
Comment (kingherc) [reply Sponsors.md]: Merged

**Review by kingherc — COMMENTED** [review]
@mzampetakis thanks for the feedback. Please review again.
@webdevgr , @nickorfas , feel free as well to review.

File: Sponsors.md
Comment (nickorfas) [inline Sponsors.md:10]: I think that it should be mentioned what other benefits sponsors get when sponsoring via open collective. For example, they are promoted via social media announcements prior to the meetup and also on post-meetup thank you messages.

They are also included (but we should decide on when) on [this](https://www.devstaff.gr/sponsors) page of our website.

File: Sponsors.md
Comment (nickorfas) [inline Sponsors.md:34]: Shouldn't we align the prices according to the latest updates of open collective sponsorship options?

Comment (mzampetakis) [reply Sponsors.md:34]: Yes I agree on this.

File: Sponsors.md
Comment (mzampetakis) [reply Sponsors.md:10]: If I can recall correctly we only mention them, after the meetup.

Comment (mzampetakis) [reply Sponsors.md:10]: I suppose that the complete list is this:
```
* leaflets (can be left on a table at entrance),
* online banner on DevStaff website (with back-link do-follow),
* "Thank you" message by organizers during Meetup (not by company),
* Sponsor's slide in DevStaff Welcome presentation,
* "Thank you" mention in our social media.
```

Comment (kingherc) [reply Sponsors.md]: Thanks! I made a new section "Awareness benefits" that tries to gather the standard ways we promote a sponsor. Please review.

File: Sponsors.md
Comment (kingherc) [reply Sponsors.md]: Makes sense, done!

**Review by kingherc — COMMENTED** [review]
Thanks @nickorfas ! Please review again. cc @m-zampetakis

**Review by mzampetakis — DISMISSED** [review]
Great work @kingherc !! 👏

File: Sponsors.md
Comment (nickorfas) [reply Sponsors.md]: Correct, social media include sponsors on post-meetup "thank you" posts.

Apart from the list that you mentioned @mzampetakis , I think that we also include the sponsors of the monthly meetup in meetup.com announcement (if we want to make it more explicit on what sponsors get as a benefit and also be more precise on what refers to the meetup time range and what is available online before and after the meetup).

So, as a conclusion, I would prefer to have a complete list of what we offer as a return of a sponsorship instead of just mentioning `we will feature you in those meetups by mentioning you and your contribution.`

File: Sponsors.md
Comment (kingherc) [inline Sponsors.md:14]: Thanks all! @nickorfas , @mzampetakis , I made a new section listing the standard awareness benefits here. Includes the new `"Thank you" mention in the announcement of the sponsored meetups.` you mentioned.

Note that the "Catering Sponsor" and "Venue Sponsor" sections mention two additional benefits (that I kept from before this PR): online banner & slide.

**Review by kingherc — COMMENTED** [review]
Thanks all! Please give any last feedback, else I'll merge this soon.

**Comment by kingherc** [comment]
Ah I see the reviews are dismissed automatically when I make a new commit. So @mzampetakis , @nickorfas , I need one approval at least :)

**Review by mzampetakis — APPROVED** [review]

File: Sponsors.md
Comment (nickorfas) [inline Sponsors.md:22]: I think that this should be rephrased to something like `The sponsor will be considered to be added in our website's sponsor page`

Comment (kingherc) [reply Sponsors.md]: Done!

**Review by kingherc — COMMENTED** [review]
Thanks @nickorfas , changed. Please review :)

**Comment by nickorfas** [comment]
> Thanks @nickorfas , changed. Please review :)

Thanks a lot for the effort on our Sponsors.md. Great job!

I am approving but please wait for an approval from @mzampetakis as well (since he also had some comments that need to be resolved).

**Review by nickorfas — APPROVED** [review]

**Comment by kingherc** [comment]
He approved before your last comment, so I think he's fine because I took care of his comments. It's just that with every commit, the approvals get dismissed and need to be done again (see above the github actions). So unless I get more feedback, I'll merge this later today.

**Comment by nickorfas** [comment]
> He approved before your last comment, so I think he's fine because I took care of his comments. It's just that with every commit, the approvals get dismissed and need to be done again (see above the github actions). So unless I get more feedback, I'll merge this later today.

Sorry, I missed that he had approved previously. You are good to go 🚀

**Pull request merged**
